tests/phpunit/testdata/plugins/test-plugin-plugin-readme-errors-license/readme.txt

```diff
@@ -0,0 +1,11 @@
+
+=== Test Plugin with readme ===
+
+Contributors:      plugin-check
+Requires at least: 6.0
+Tested up to:      6.1
+Requires PHP:      5.6
+Stable tag:        1.0.0
+Tags:              testing, security
```

Comment:
It would be better to include an _invalid_ license entry here rather than none. That's because ensuring that the regex works is probably more important than to validate existence of the entry, the latter of which would be far easier to check (without a regex).

Reply:
Address in https://github.com/10up/plugin-check/pull/191/commits/e832dfce38d83d396351736be2b0faec5e2eefd8